Assets/MRTK/Providers/XRSDK/Profiles/DefaultXRSDKInputSystemProfile.asset

```diff
@@ -21,6 +21,13 @@ MonoBehaviour:
     priority: 0
     runtimePlatform: 9
     deviceManagerProfile: {fileID: 0}
+  - componentType:
+      reference: Microsoft.MixedReality.Toolkit.XRSDK.Oculus.OculusXRSDKDeviceManager,
+        Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus
+    componentName: XRSDK Oculus Device Manager
+    priority: 0
+    runtimePlatform: 33
```

Comment:
We may need to add some data provider guards around XR platform soon, since both this and the WMR device manager will run on Windows standalone. We'll likely get duplicate controllers in that case.

I'm currently investigating a better way to take advantage of the cross-platform nature of XR SDK with our data providers (in theory, we should really only need one!). Mostly commenting here for info, and less for any action in this PR.